.github/workflows/manual-publish-formula-update.yml

```
@@ -0,0 +1,61 @@
+name: Manual update Product Formula in Tap
+# In the case of automation failure, this workflow allows a user to manually run a product version update.
+on:
+    workflow_dispatch:
+      inputs:
+        product:
+          description: product name as it appears in binary (ie. "vault", "vault-enterprise", etc.)
+        version:
+          description: semantic version string (ie. "1.12.2")
+        is_cask:
+          type: boolean
+          description: check if updating a cask (rather than a tap formula)
```

```suggestion
          type: boolean
          description: check if updating a cask (rather than a tap formula)
          default: false
```

Mostly because I have no idea what the default behavior in GHA is for a boolean, but we have fewer Casks than Formulas so `false` it is!